axelor-base/src/main/java/com/axelor/apps/base/service/AddressServiceImpl.java

```diff
@@ -270,31 +285,11 @@ public String computeFullName(Address address) {
 
   @Override
   public String computeAddressStr(Address address) {
-    StringBuilder addressString = new StringBuilder();
-    if (address == null) {
-      return "";
-    }
-
-    if (StringUtils.notBlank(address.getAddressL2())) {
-      addressString.append(address.getAddressL2()).append(System.lineSeparator());
-    }
-    if (StringUtils.notBlank(address.getAddressL3())) {
-      addressString.append(address.getAddressL3()).append(System.lineSeparator());
-    }
-    if (StringUtils.notBlank(address.getAddressL4())) {
-      addressString.append(address.getAddressL4()).append(System.lineSeparator());
+    try {
+      setFormattedFullName(address);
+    } catch (AxelorException e) {
```

Comment:
You have to delete this. You told me you kept it because during the demo data import you had some errors but it's because the formattedFullName is null on all the imported addresses.
So if you fix the problem that is making the formattedFullName on all the impoted addresses, then you won't have other problems after.
So delete this code please.